public/static/docs/command-reference/get.md

```diff
@@ -181,3 +188,16 @@ Untracked files:
 	model.bigrams.pkl
 	model.monograms.pkl
 ```
+
+### Example: Getting a url for a file in a DVC repository
+> Note that `dvc get` can be used from anywhere in the file system, as long as
+> DVC is [installed](/doc/install).
+
+We can use `dvc get --show-url` to get the actual location of the resulting model file from our
+[get started example repo](https://github.com/iterative/example-get-started), a
+<abbr>DVC project</abbr> hosted on GitHub:
```

Comment:
Suggestion, not necessarily to use exactly:

``We can use `dvc get --show-url` to get the actual location where the final model file from our [get started example repo](https://github.com/iterative/example-get-started) is stored. Note that `remote.dvc.org/get-started` is an HTTP [DVC remote](/doc/command-reference/remote), whereas `662eb7f64216d9c2c1088d0a5e2c6951` is the file's checksum.``

Reply:
Updated in 663f589a.